checks/rolebindingClusterRolePodExecAttach.yaml

```diff
@@ -0,0 +1,94 @@
+successMessage: The RoleBinding does not reference a ClusterRole allowing pods/exec or pods/attach
+failureMessage: The RoleBinding references a ClusterRole that allows Pods/exec, allows pods/attach, or that does not exist
```

Comment:
ditto

Reply:
I believe this message is correct, because the primary schema looks at bindings, and the additional schema looks at the ClusterRole associated with the binding.